screens/WordRoulette.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react'
+import { Image, Linking } from 'react-native';
+import { Container, Header, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body, Right } from 'native-base';
+import { NavigationActions } from "react-navigation";
+
+export default class WordRoulette extends Component {
```

Comment:
Is this tested? It looks like JQuery, is that supported by React Native?

Reply:
It's for navigating through react. it was used on the initial starter code

Reply:
Oh ya you're right I was supposed to comment that out.